### DISCUSSION TITLE
How to delete history of searching?

### DISCUSSION THREAD

**Discussion by mariush444** [osmandapp/OsmAnd#19092] [Q&A]
I deleted history in oand settings menu but they are still exists.

Osmamd 4.5.10

<table style="width:100%">
<tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/66887280/e806b8f7-2d9b-4913-bfba-0b8af892452f"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/66887280/d1fb021d-2b7e-4513-b72f-d3ee4d658b5b"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by DmitryAlexei** [comment]
Take a look at the navigation search and (if needed) delete it. Related issue was fixed in the release version from Google play.
